kao.sty

```diff
@@ -1173,12 +1169,14 @@
 
 % Choose the default colors
 \hypersetup{
+	%allcolors=DarkGreen,
 	%anchorcolor=Red,
 	%citecolor=DarkOrange!70!black,
 	citecolor=OliveGreen,
 	filecolor=OliveGreen,
 	%linkcolor=Blue,
 	linkcolor=Black,
```

Comment:
Also, this line seems to force a reference in a colored text, i.e., `\textcolor{red}{... \ref{key} ...}`, to be printed in black. It might be better not specifying it.
```suggestion
	linkcolor=,
```

Reply:
Good points, I somehow missed the redundancy.